doc/dev-status.rst

```diff
@@ -22,6 +22,8 @@ Last updated: ``2017-12-14``
 +--------------------------------------+--------+--------+--------+--------+--------------+--------+--------+--------+----------+
 |`py-autocrypt`_                       |✔       |✔       |✔       |✔       |✘             |✘       |✘       |✘       |✔         |
 +--------------------------------------+--------+--------+--------+--------+--------------+--------+--------+--------+----------+
+|`pyac`_                               |✔       |✔       |✔       |✔       |✔             |✔       |✔       |✔       |✔         |
```

Comment:
wow, surprised to see all checkmarks already :)  curious: How does pyac generate recommendations?  couldn't find something dealing with it in the source (but may have missed it) Can pyac already process ASMs from delta or enigmail?  And do you get gossip keys from the encrypted parts of emails?

Reply:
> wow, surprised to see all checkmarks already :) curious: How does pyac generate recommendations? 

sorry, but copy/paste, fixed

> Can pyac already process ASMs from delta or enigmail?

i haven't checked, feel free to check it.
Maybe we should also add those to the tests in autocrypt repo?

> And do you get gossip keys from the encrypted parts of emails?

Yes: juga0/pyac/blob/master/autocrypt/message.py#L332